test/validator-info.test.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import nacl from 'tweetnacl';
+
+import {PublicKey} from '../src/publickey';
+import {ValidatorInfo} from '../src/validator-info';
+
+test('from config account data', () => {
+  const keypair = nacl.sign.keyPair.fromSeed(
+    Uint8Array.from(Array(32).fill(8)),
+  );
+
+  const expectedValidatorInfo = new ValidatorInfo(
+    new PublicKey(keypair.publicKey),
+    {
+      name: 'Validator',
+      keybaseId: 'validator_id',
+    },
+  );
+
+  const configData = Buffer.from(
+    'AgdRlwF0SPKsXcI8nrx6x4wKJyV6xhRFjeCk8W+AAAAAABOY9ixtGkV8UbpqS189vS9p/KkyFiGNyJl+QWvRfZPKAS8AAAAAAAAAeyJrZXliYXNlSWQiOiJ2YWxpZGF0b3JfaWQiLCJuYW1lIjoiVmFsaWRhdG9yIn0',
```

Comment:
Cool, can you sneak in a comment that documents how you generated this *random* string for the poor soul that needs to update it if/when the format changes :)

Reply:
hehe, will do :)